.github/workflows/ci.yml

```diff
@@ -0,0 +1,156 @@
+# CI WorkFlow for build, test and release of Plugin Packages
+name: CI for Plugin Packages
+on:
+  push:
+    branches:
+      - 'main'
+    tags:
+      - 'v*.*.*'
+    paths-ignore:
```

Comment:
remove path filters for `tag` release flow

Reply:
yes.. done. Please review